packages/gestalt/src/PageHeader.flowtest.js

```diff
@@ -0,0 +1,8 @@
+// @flow strict
+import React from 'react';
+import PageHeader from './PageHeader.js';
+
+const Valid = <PageHeader title="Settings" />;
```

Comment:
how can it be Valid if there's no title prop?

![image](https://user-images.githubusercontent.com/10593890/111223331-611f1800-85b3-11eb-94db-75889b94d176.png)